kms/auth-eth/contracts/AppAuth.sol

```diff
@@ -131,11 +125,6 @@ contract AppAuth is
     function isAppAllowed(
         IAppAuth.AppBootInfo calldata bootInfo
     ) external view override returns (bool isAllowed, string memory reason) {
-        // Check if this controller is responsible for the app
-        if (bootInfo.appId != appId) {
-            return (false, "Wrong app controller");
-        }
-
```

Comment:
We also need to remove `appId` from `AppBootInfo`?

https://github.com/Dstack-TEE/dstack/blob/e26d73d9bf7c3ef8010395cc7220187a2b3fe748/kms/auth-eth/contracts/IAppAuth.sol#L32